### ISSUE TITLE
Switching between portrait and landscape orientation UI stucks between two tabs

### ISSUE THREAD

**Issue by Thebarda** [satya164/react-native-tab-view#877]
### Current behaviour

Especially on scrollable tab, when i'm switching between vertical and horizontal device orientation, UI stucks between two tabs but it could be unblocked by a swipe or a press on a tab item

### Expected behaviour

when i'm switching between vertical and horizontal device orientation, UI should stay at same tab 

### Code sample

For Syntax Highlighting check this [link](https://help.github.com/en/articles/creating-and-highlighting-code-blocks)

There is a repro in [example](https://github.com/react-native-community/react-native-tab-view/tree/master/example) folder

### Screenshots (if applicable)

![Screenrecorder-2019-09-09-16-41-17-382](https://user-images.githubusercontent.com/12515407/64541277-e3c17a00-d321-11e9-9c5d-cc08297774b7.gif)
Sorry for bad quality :/

### What have you tried

Nothing for the moment

### Your Environment

| software                     | version
| ---------------------------- | -------
| android               | 28
| react-native                 | 0.59.8 (same bug with 0.60.4)
| react-native-tab-view        | 
| react-native-gesture-handler | 1.2.1
| react-native-reanimated      | 1.0.1
| node                         | 10.15.2
| npm                  | 6.9.0

**Comment by deefourple** [comment]
@Thebarda Any luck on resolving this? I'm getting the same issue for both Android & iOS.

**Comment by Thebarda** [comment]
@deefourple Sorry but I'm in vacation until next week. I'll keep you in touch

**Comment by wayne214** [comment]
sorry, the problem still exists

**Comment by Thebarda** [comment]
@wayne214 I've just create another PR

**Comment by truongnv91** [comment]
Same error

**Comment by osdnk** [comment]
I made some hack in order to make it workable. Idk if it helps your cases #915

**Comment by jsamr** [comment]
@satya164 @osdnk I've had multiple users reporting this issue, mainly on iOS, with version 2.10.2... I think you can reopen.

**Comment by Pipo93** [comment]
The problem still exists with current version (2.13.0) in our case... Should be reopened since the fix of #915 was reverted in November

**Comment by karmil32** [comment]
any update? i still facing this issue ...

**Comment by Jarkendar** [comment]
The problem still exists.

**Comment by rubita-svmx** [comment]
I have the same problem but only in ios.

**Comment by alvaro1728** [comment]
Same problem.  It happens when I'm on one of the inner tabs and I switch from portrait to landscape or viceversa.  It's happening on both iOS and Android for me.

**Comment by alvaro1728** [comment]
Found a workaround by forcing the component to refresh itself when the orientation changes. 

    const window = Dimensions.get("window");

    return (
      <Tab.Navigator
        key={window.width}  
        ...

Cheers,
Alvaro

**Comment by Knight704** [comment]
The problem is also present in @react-navigation/material-top-tabs since it is based on react-native-tab-view

Any suggestion on that please?

**Comment by alvaro1728** [comment]
Updating `react-native-tab-view` to the latest version (3.0.0) solved this for me.

**Comment by satya164** [comment]
Closing since this should be fixed in latest version.

**Comment by summer-wu** [comment]
I've tested, these two combination works fine.
```
"react-native-pager-view": "5.4.25
"react-native-tab-view": "3.3.0"
```
And these two combination works bad as the poster says:
```
"react-native-pager-view": "5.4.14"
"react-native-tab-view": "3.3.0"
```
